package.json

```diff
@@ -151,13 +151,11 @@
             ],
             "examples": [
               {
-                "serverUrl": "https://<replace_with_your_sonarqube_server_url>",
```

Comment:
Good point, and since the UI has issues in Codespaces, putting the token in the settings and letting the migration run is one possible workaround to configure connected mode.